operator/src/main/resources/scripts/startServer.sh

```diff
@@ -126,6 +132,11 @@ ${SCRIPTPATH}/wlst.sh $SCRIPTPATH/start-server.py
 # Wait forever.   Kubernetes will monitor this pod via liveness and readyness probes.
 #
 
-trace "Wait indefinitely so that the Kubernetes pod does not exit and try to restart"
-while true; do sleep 60; done
+if [ "${SERVER_OUT_IN_POD_LOG}" == 'true' ] ; then
+  trace "Showing the server out file from ${SERVER_OUT_FILE}"
+  tail -F -n +0 ${SERVER_OUT_FILE}
```

Comment:
What happens if the tail command fails?    Maybe we need an "exit 1" right after.